src/Dockerfile

```diff
@@ -1,20 +1,28 @@
-FROM centos:centos8.3.2011
+FROM centos:centos8.4.2105
 
 LABEL org.opencontainers.image.authors="Adfinis AG <https://adfinis.com>"
 LABEL org.opencontainers.image.vendor="Adfinis"
 
-RUN yum install -y \ 
-device-mapper-multipath \ 
-iscsi-initiator-utils \ 
-nvme-cli \
-xfsprogs \ 
-nfs-utils \ 
-net-tools \ 
-e2fsprogs 
+ENV HUAWEI_ESDK_K8S_PLUGIN_VERSION=V2.2.13
 
-ADD ["bin/huawei-csi", "/"] 
-ADD ["bin/passwdEncrypt", "/"] 
-RUN ["chmod", "+x", "/huawei-csi"] 
-RUN ["chmod", "+x", "/passwdEncrypt"] 
+RUN yum install -y \
+    device-mapper-multipath \
+    iscsi-initiator-utils \
+    nvme-cli \
+    xfsprogs \
+    nfs-utils \
+    net-tools \
+    e2fsprogs \
+    unzip \
```

Comment:
can you sort these by alphabet?

Reply:
Those are sorted according to the Huawei eSDK documentation.

Reply:
best reference implementation evar ;)